crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -43,14 +43,6 @@ module sui::sui_system {
     // - the change in the validator set across epochs can be at most x validators
     //
     struct SystemParameters has store {
-        /// Lower-bound on the amount of stake required to become a validator.
-        // TODO: use a different threshold for new validator joining vs. validator keeping their
```

Comment:
are we still planning on using a different threshold for new validators?

Reply:
Yes. I'm doing that in a separate PR and it will be another constant.